simpleviewinc/tap-events-force
# Zen 631 Mobile Drawer Icons

**Ticket**: [ZEN-631](https://jira.simpleviewtools.com/browse/ZEN-631)

## Context
* Updated the drawer to match the new Events-Force mock-ups

## Updates
* `src/assets/icons/evf/elements/*`
  * Add Plus and Minus icons
* `src/components/grid/gridRowContent.js`
  * Switch the Drawer to use the Chevron Icon
* `src/components/textToggle/evfTextToggle.js`
  * Update to use the Plus and Minus Icons
  * Updated name of hook

## Tests
**Setup**
* Run the image =>  `keg evf pack run evf:zen-631-mobile-drawer`
* Navigate it in your browser => http://evf-zen-631-mobile-drawer.local.keghub.io/


**Test 1**
* Ensure the chevron icon displays properly on each session item
  * [Looks like this](https://i.imgur.com/cf3L1lN.png)
* Ensure it matches the mock-ups
  * [Looks like this](https://i.imgur.com/cBtC1nq.png)

**Test 2**
* Ensure the Plus and Minus icons displays properly on each session item
  * [Looks like this](https://i.imgur.com/FhMA9lK.png)
* Ensure it matches the mock-ups
  * [Looks like this](https://i.imgur.com/pH7lyEX.png)
